reactions/src/SpecialFunctions.cpp

```diff
@@ -451,11 +451,6 @@ int funcn1(double a, const double G[], double F[], void *params)
 	mu2 = mu(a,k2,omega0,p1,p2,p3);
 	mu3 = mu(a,karg1,omega0,p1,p2,p3);
 
```

Comment:
Why no mu4=mu(a,karg4,omega0,p1,p2,p3); to use in F[9] as in funcn1_psuedo?

Reply:
It was added explicitly into F[9] - have changed it to be consistent with funcn1_pseudo.